api/datastore/mock.go

```diff
@@ -315,7 +325,7 @@ func (s sortF) Len() int           { return len(s) }
 func (s sortF) Less(i, j int) bool { return strings.Compare(s[i].Name, s[j].Name) < 0 }
 func (s sortF) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
 
-func (m *mock) GetFns(ctx context.Context, filter *models.FnFilter) ([]*models.Fn, error) {
+func (m *mock) GetFns(ctx context.Context, filter *models.FnFilter) (*models.FnList, error) {
 	// sort them all first for cursoring (this is for testing, n is small & mock is not concurrent..)
 	sort.Sort(sortF(m.Fns))
```

Comment:
does the fn cursor not get base64'd ?

Reply:
looks like it doesn't... it certainly should!